src/pages/components/components.json

```
@@ -1,4 +1,16 @@
 [
+  {
+    "title": "sveltefire",
+    "url": "https://github.com/codediodeio/sveltefire",
+    "npm": "https://www.npmjs.com/package/sveltefire",
```

This should be the NPM package name, not the URL to the NPM page
```suggestion
    "npm": "sveltefire",
```